tox.ini

```diff
@@ -19,7 +19,7 @@ setenv =
 deps =
   -rrequirements/base.txt
   -rrequirements/test.txt
-commands = pytest {posargs:} --cov zoo test
+commands = pytest {posargs:} --cov zoo test --cov-report=term:skip-covered
```

Comment:
Sorry, it’s not related. I’ll remove it.  
It’s to avoid having a too long list of 100% covered files in the output, so we can concentrate on the one not covered.